cypress/e2e/patient_spec/PatientConsultationDischarge.cy.ts

```diff
@@ -76,10 +76,8 @@ describe("Patient Discharge based on multiple reason", () => {
     patientDischarge.clickClearButton();
     // select a non-registered facility and perform the discharge
     patientDischarge.typeReferringFacility(referringFreetextFacility);
-    cy.wait(2000);
     cy.clickSubmitButton("Confirm Discharge");
     cy.clickSubmitButton("Acknowledge & Submit");
-    cy.wait(2000);
```

Comment:
- since you are removing all wait, add a intercept to verify the discharge is successful